src/lsp/sql_parser/grammar.mly

```diff
@@ -0,0 +1,134 @@
+%{
+(**************************************************************************)
+(*                                                                        *)
+(*                        SuperBOL OSS Studio                             *)
+(*                                                                        *)
+(*  Copyright (c) 2022-2023 OCamlPro SAS                                  *)
+(*                                                                        *)
+(* All rights reserved.                                                   *)
+(* This source code is licensed under the GNU Affero General Public       *)
+(* License version 3 found in the LICENSE.md file in the root directory   *)
+(* of this source tree.                                                   *)
+(*                                                                        *)
+(**************************************************************************)
+open Sql_ast
+%}
+%token EOF
+%token EXEC SQL END_EXEC
+%token BEGIN END DECLARE SECTION DISCONNECT COMMA LPAR RPAR DOT
+(*For whenever*)
+%token WHENEVER NOT FOUND SQLERROR SQLWARNING CONTINUE PERFORM GOTO
+(*For include*)
+%token INCLUDE SQLCA SQLDA
+(*For Connect*)
+%token CONNECT TO USER WITH NO PROMPT RETURNING IDENTIFIED BY AT USING RESET DSN AS
+(*For Transaction*)
+%token START TRANSACTION
+(*For Rollback*)
+%token ROLLBACK WORK RELEASE SAVEPOINT 
+(*For commit*)
+%token COMMIT
+(*For SQL*)
+%token ASTERIX SELECT FROM
+%token <string> WORD
+%token <string> STRING
+%token <string> DIGITS
+%token <string> COBOL_VAR
+%token <string> BACKSLASHED_VAR
+%start <esql_instuction> main 
+
+%%
+
+let main :=
+| EXEC; SQL; stm=esql ; END_EXEC; EOF; {stm}
+
+let variable := 
+| s=WORD; {SqlVar s} 
+| s=COBOL_VAR; {CobolVar s} 
+
+let esql := 
+| AT; v=variable; stm=esql; {At(v, stm)}
```

Comment:
To add the location to `v` use the `loc` rule (copied from `/src/lsp/cobol_parser/grammar.mly`).

Reply:
I must have done something wrong with the toml package, because i have a error when i try to compile with these changes

`Error: Unbound module Cobol_parser.Grammar_utils`

Reply:
You should probably add this to the `dependencies` section of `package.toml` :
```
cobol_parser = "version"
```
Then call `drom project` (or `opam exec -- drom project)` to regenerate the files.

Reply:
I've already try :(